rsts/getting_started/index.rst

```diff
@@ -142,6 +142,5 @@ To summarize, you have:
     (b) on a demo Flyte cluster.
 
 What's Next?
-^^^^^^^^^^^^^^^^
-
-To experience the full power of Flyte, take a look at the `User Guide <https://docs.flyte.org/projects/cookbook/en/latest/user_guide.html>`__.
+^^^^^^^^^^^^^^^
+This guide demonstrated how you can quickly iterate on self-contained scripts using ``pyflyte run``. This command combined a few steps into one. For larger projects, you will need to separately package your code, register those compiled objects with Flyte, and launch your tasks and workflows. To experience the full power of Flyte, please follow to guide for :ref:`working with repos <cookbook:larger_apps>` as well as the complete set of examples in the  `User Guide <https://docs.flyte.org/projects/cookbook/en/latest/user_guide.html>`__.
```

Comment:
my 2 cents:
This is a lot of information to be presenting here towards the end of this getting started guide... if I were a new user, I'd want to see more of the features of Flyte (aka the User Guide) before deciding I want a full-blown project.

Suggestion: Use a bullet point-list to provide a fork in the road:
```
This guide demonstrated how you can quickly iterate on self-contained scripts using ``pyflyte run``.

- To learn more about Flyte's features such as caching, conditionals, and scheduling workflows, take a look at the `User Guide <https://docs.flyte.org/projects/cookbook/en/latest/user_guide.html>`__.
- To learn more about how to organize larger projects, see the guide for :ref:`Building Large Apps <cookbook:larger_apps>`.
```

Reply:
I agree this is too verbose. 
Lets do this, on the top of this file, we can say get started with monolith python files?

Reply:
> monolith python files

to push back on this a little, I think we need to slowly introduce use cases... the purpose of getting started is to quickly run a workflow on a workflow cluster, so I don't think there's a need introduce the idea of monolith python files.

We should definitely add a link to "Building Large Apps" at the end of this guide. I'll add these changes to https://github.com/flyteorg/flyte/pull/2370